bitbucket-pipelines.yml

```diff
@@ -5,16 +5,44 @@ definitions:
     docker:
       memory: 3072
   steps:
-    - step: &test
-        name: Test
+    - step: &install
+        name: ⚙️ Install
         script:
-          - FORCE_COLOR=true
           - npm ci
+        artifacts:
+          - node_modules/**
+    - step: &lint
+        name: 🧹 Lint
+        script:
           - npm run lint
-          - npm run check-types
+    - step: &test
+        name: 🧪 Test
+        script:
           - npm run test
+    - step: &deploy-temp-stack
+        name: 🏗️ Test Deployment
+        script:
+          - TMP_STAGE=$(cat /dev/urandom | tr -dc 'a-z' | head -c3 | grep -vE 'prd|stg')
+          - echo "Deploy temporary service to stage:\ $TMP_STAGE"
+          - pipe: docker://aligent/serverless-deploy-pipe:latest-node18
+            variables:
+              AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+              AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+              CFN_ROLE: ${CFN_ROLE}
+              DEBUG: ${CI_DEBUG}
+              STAGE: ${TMP_STAGE}
+          - pipe: docker://aligent/serverless-deploy-pipe:latest-node18
```

Comment:
Can  [this](https://github.com/aligent/serverless-deploy-pipe/blob/bd16a93a62e47ce370c971d5a314a0118620141a/pipe/pipe.py#L240)  remove the resources like non-empty S3 buckets?

Reply:
I believe so, yes :pray: